### PR TITLE
Adding method to return full name of appointment

### PR DESCRIPTION
**Pull request by Guntrisoft**
- output the full name in breadcrumb and title to help
  user know what they are edit/rescheduling

### CODE REVIEW

File: spec/models/appointment_spec.rb [hunk 1]
```diff
@@ -187,4 +187,14 @@ def results(q, start_at, end_at)
       expect(results).to eq [appointment]
     end
   end
+
+  describe '#name' do
+    let(:subject) do
```
Comment (benlovell) [inline spec/models/appointment_spec.rb]: You can just use `subject { ... }` here instead.

File: app/models/appointment.rb [hunk 2]
```diff
@@ -42,6 +42,10 @@ def self.full_search(query, start_at, end_at)
     results
   end
 
+  def name
+    [first_name, last_name].join(' ')
```
Comment (benlovell) [inline app/models/appointment.rb]: Interpolating this would be more concise.

File: spec/models/appointment_spec.rb (hunk 1 above)
Comment (Guntrisoft) [inline spec/models/appointment_spec.rb]: done

File: app/models/appointment.rb (hunk 2 above)
Comment (Guntrisoft) [inline app/models/appointment.rb]: done

**Comment by benlovell** [comment]
:100:

**Pull request merged**